.github/workflows/apidocs-upload.yml

```diff
@@ -0,0 +1,85 @@
+name: apidocs
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+
+  # First check if this version is a valid public release
+  check-version:
+    runs-on: ubuntu-20.04
```

Comment:
Should we use a newer version of ubuntu?

Reply:
Updated on [f3a3fd5](https://github.com/HathorNetwork/hathor-wallet-headless/pull/364/commits/f3a3fd5fef5e52c5702397a7ebabdc1ea5832284)